sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -165,18 +166,30 @@ def get_next(next_link=None):
         return ItemPaged(get_next, extract_data)
 
     @distributed_trace
-    def get_repository_client(self, repository, **kwargs):
-        # type: (str, Dict[str, Any]) -> ContainerRepositoryClient
-        """Get a repository client
+    def get_repository(self, repository, **kwargs):
+        # type: (str, Dict[str, Any]) -> ContainerRepository
+        """Get a Container Repository object
 
         :param str repository: The repository to create a client for
```

Comment:
This is documented as a `str`, though type-hint says it can also be a `dict` - is that why it wasn't changed to `repository_name` like the others?

Reply:
This is an inconsistency on my part. The param should be `repository_name` and it will be a str. The type-hint is meant to be: `def get_repository(self, repository: str, **kwargs: Any) -> ContainerRepository: